### PR TITLE
Feature/add host container mapping params

### PR DESCRIPTION
**Pull request by thanset**
* Add parameters for host and container path settings for the broker executors.

### CODE REVIEW

**Comment by thanset** [comment]
@aaronjwood Could you please take a look when you have a moment? Thanks.

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Wed Apr 06 16:42:39 CDT 2016
+#Thu Oct 20 15:26:46 EDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.12-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.12-all.zip
```
Comment (aaronjwood) [inline gradle/wrapper/gradle-wrapper.properties]: What's this do?

File: kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/offer/PersistentOfferRequirementProvider.java [hunk 2]
```diff
@@ -452,6 +458,17 @@ private ExecutorInfo getNewExecutorInfo(KafkaSchedulerConfiguration config, Stri
...
 
+    private ContainerInfo getNewContainer(String hostPath, String containerPath){
+        return org.apache.mesos.Protos.ContainerInfo.newBuilder()
+                .addVolumes(org.apache.mesos.Protos.Volume.newBuilder()
+                .setContainerPath(containerPath)
+                .setHostPath(hostPath)
+                .setMode(Volume.Mode.RW)
+                .build())
+                .setType(ContainerInfo.Type.MESOS)
+                .build();
```
Comment (aaronjwood) [inline kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/offer/PersistentOfferRequirementProvider.java]: Do you need to `build()` twice?

Comment (thanset) [inline kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/offer/PersistentOfferRequirementProvider.java]: Yes, the builder for the Volume, then the ContainerInfo.  There's no method to take in a Volume.Builder param for the addVolumes() method, only a Volume.

**Pull request merged**